### PR TITLE
Eagle

### PR DESCRIPTION
**Pull request by fsansalvadore**
https://zone-www-dot-com-git-feat-lw11-ticketing-supabase.vercel.app/special-announcement

### CODE REVIEW

File: apps/www/pages/launch-week/index.tsx [hunk 1]
```diff
@@ -1,36 +1,21 @@
 import { useState, useEffect } from 'react'
```
[the remark this answers is not in the document]
Comment (fsansalvadore) [reply apps/www/pages/launch-week/index.tsx:1]: ?

File: apps/www/lib/constants.ts [hunk 2]
```diff
@@ -40,8 +40,10 @@ export const LW8_DATE = 'August 7-11'
 export const LW8_LAUNCH_DATE = '2023-08-07T09:00:00.000-07:00'
 export const LWX_DATE = 'Dec 11-15 / 8am PT'
 export const LWX_LAUNCH_DATE = '2023-12-11T08:00:00.000-08:00'
-export const TWEET_TEXT =
-  '#SupaLaunchWeek 🆇 is coming December 11-15 🚀\n\nClaim your ticket for a chance to win @supabase swag:\n'
-export const TWEET_TEXT_GOLDEN = `I turned my #SupaLaunchWeek 🆇 ticket Platinum! 🩶😎\n\nClaim your own ticket to win @supabase swag:\n`
-export const TWEET_TEXT_SECRET = `I found a secret #SupaLaunchWeek 🆇 ticket! \n\nGuess the word at supabase.com/launch-week to get yours:\n`
+export const LW11_DATE = 'April 15-19 / 7am PT'
+export const LW11_LAUNCH_DATE = '2024-04-15T07:00:00.000-08:00'
+export const TWEET_TEXT = 'Claim your ticket to join a @supabase Special Event'
```
Comment (awalias) [inline apps/www/lib/constants.ts:45]:
```suggestion
export const TWEET_TEXT = 'Claim your ticket to join this @supabase Special Event'
```

File: apps/www/lib/constants.ts [hunk 3]
```diff
@@ -40,8 +40,10 @@ export const LW8_DATE = 'August 7-11'
 export const LW8_LAUNCH_DATE = '2023-08-07T09:00:00.000-07:00'
 export const LWX_DATE = 'Dec 11-15 / 8am PT'
 export const LWX_LAUNCH_DATE = '2023-12-11T08:00:00.000-08:00'
-export const TWEET_TEXT =
-  '#SupaLaunchWeek 🆇 is coming December 11-15 🚀\n\nClaim your ticket for a chance to win @supabase swag:\n'
-export const TWEET_TEXT_GOLDEN = `I turned my #SupaLaunchWeek 🆇 ticket Platinum! 🩶😎\n\nClaim your own ticket to win @supabase swag:\n`
-export const TWEET_TEXT_SECRET = `I found a secret #SupaLaunchWeek 🆇 ticket! \n\nGuess the word at supabase.com/launch-week to get yours:\n`
+export const LW11_DATE = 'April 15-19 / 7am PT'
+export const LW11_LAUNCH_DATE = '2024-04-15T07:00:00.000-08:00'
+export const TWEET_TEXT = 'Claim your ticket to join this @supabase Special Event'
```
Comment (awalias) [inline apps/www/lib/constants.ts:45]:
```suggestion
export const TWEET_TEXT = ''
```

File: apps/www/lib/constants.ts [hunk 4]
```diff
@@ -40,8 +40,10 @@ export const LW8_DATE = 'August 7-11'
 export const LW8_LAUNCH_DATE = '2023-08-07T09:00:00.000-07:00'
 export const LWX_DATE = 'Dec 11-15 / 8am PT'
 export const LWX_LAUNCH_DATE = '2023-12-11T08:00:00.000-08:00'
-export const TWEET_TEXT =
-  '#SupaLaunchWeek 🆇 is coming December 11-15 🚀\n\nClaim your ticket for a chance to win @supabase swag:\n'
-export const TWEET_TEXT_GOLDEN = `I turned my #SupaLaunchWeek 🆇 ticket Platinum! 🩶😎\n\nClaim your own ticket to win @supabase swag:\n`
-export const TWEET_TEXT_SECRET = `I found a secret #SupaLaunchWeek 🆇 ticket! \n\nGuess the word at supabase.com/launch-week to get yours:\n`
+export const LW11_DATE = 'April 15-19 / 7am PT'
+export const LW11_LAUNCH_DATE = '2024-04-15T07:00:00.000-08:00'
+export const TWEET_TEXT = ''
```
Comment (awalias) [inline apps/www/lib/constants.ts:45]:
```suggestion
export const TWEET_TEXT = '👁️⚡👁️'
```

File: apps/www/lib/constants.ts [hunk 5]
```diff
@@ -40,8 +40,10 @@ export const LW8_DATE = 'August 7-11'
...
 export const LWX_LAUNCH_DATE = '2023-12-11T08:00:00.000-08:00'
-export const TWEET_TEXT =
-  '#SupaLaunchWeek 🆇 is coming December 11-15 🚀\n\nClaim your ticket for a chance to win @supabase swag:\n'
-export const TWEET_TEXT_GOLDEN = `I turned my #SupaLaunchWeek 🆇 ticket Platinum! 🩶😎\n\nClaim your own ticket to win @supabase swag:\n`
-export const TWEET_TEXT_SECRET = `I found a secret #SupaLaunchWeek 🆇 ticket! \n\nGuess the word at supabase.com/launch-week to get yours:\n`
+export const LW11_DATE = 'April 15-19 / 7am PT'
+export const LW11_LAUNCH_DATE = '2024-04-15T07:00:00.000-08:00'
+export const TWEET_TEXT = '👁️⚡👁️'
+export const TWEET_TEXT_GOLDEN = ``
+export const TWEET_TEXT_PLATINUM = `Just got a platinum @supabase ticket. Claim yours!`
```
Comment (awalias) [inline apps/www/lib/constants.ts:47]:
```suggestion
export const TWEET_TEXT_PLATINUM = `👁️⚡👁️`
```

File: apps/www/lib/constants.ts [hunk 6]
```diff
@@ -40,8 +40,10 @@ export const LW8_DATE = 'August 7-11'
...
 export const LWX_DATE = 'Dec 11-15 / 8am PT'
 export const LWX_LAUNCH_DATE = '2023-12-11T08:00:00.000-08:00'
-export const TWEET_TEXT =
-  '#SupaLaunchWeek 🆇 is coming December 11-15 🚀\n\nClaim your ticket for a chance to win @supabase swag:\n'
-export const TWEET_TEXT_GOLDEN = `I turned my #SupaLaunchWeek 🆇 ticket Platinum! 🩶😎\n\nClaim your own ticket to win @supabase swag:\n`
-export const TWEET_TEXT_SECRET = `I found a secret #SupaLaunchWeek 🆇 ticket! \n\nGuess the word at supabase.com/launch-week to get yours:\n`
+export const LW11_DATE = 'April 15-19 / 7am PT'
+export const LW11_LAUNCH_DATE = '2024-04-15T07:00:00.000-08:00'
+export const TWEET_TEXT = '👁️⚡👁️'
+export const TWEET_TEXT_GOLDEN = ``
```
Comment (awalias) [inline apps/www/lib/constants.ts:46]:
```suggestion
export const TWEET_TEXT_GOLDEN = `👁️⚡👁️`
```

File: apps/www/pages/launch-week/index.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (MildTomato) [reply apps/www/pages/launch-week/index.tsx:1]: /launch-week

[the remark this answers is not in the document]
Comment (MildTomato) [reply apps/www/pages/launch-week/index.tsx:1]: maybe /special-event

[the remark this answers is not in the document]
Comment (fsansalvadore) [reply apps/www/pages/launch-week/index.tsx:1]: I think /launch-week makes sense in this context.

@awalias?

File: apps/studio/public/favicon/browserconfig.xml [hunk 7]
```diff
@@ -1,2 +1,2 @@
 <?xml version="1.0" encoding="utf-8"?>
-<browserconfig><msapplication><tile><square70x70logo src="/ms-icon-70x70.png"/><square150x150logo src="/ms-icon-150x150.png"/><square310x310logo src="/ms-icon-310x310.png"/><TileColor>#ffffff</TileColor></tile></msapplication></browserconfig>
\ No newline at end of file
+<browserconfig><msapplication><tile><square70x70logo src="/favicon/ms-icon-70x70.png"/><square150x150logo src="/favicon/ms-icon-150x150.png"/><square310x310logo src="/favicon/ms-icon-310x310.png"/><TileColor>#ffffff</TileColor></tile></msapplication></browserconfig>
```
Comment (MildTomato) [inline apps/studio/public/favicon/browserconfig.xml:2]: this right?

File: apps/www/components/LaunchWeek/11/LW11Day1-v1.tsx [hunk 8]
```diff
@@ -0,0 +1,78 @@
...
+            day.shipped && 'text-foreground'
+          )}
+        >
+          {day.dd} {day.date}
+        </div>
+        <Link
+          href={day.blog}
+          className="relative group/link flex items-center gap-2 text-sm translate-x-0 !ease-[.24,0,.22,.99] duration-200 hover:-translate-x-6 transition-transform hover:cursor-pointer"
+        >
+          <IconEdit2 className="w-4 min-w-4 group-hover/link:opacity-0 transition-opacity" />
```
Comment (MildTomato) [inline apps/www/components/LaunchWeek/11/LW11Day1-v1.tsx:32]: deprecated

File: packages/ui/src/components/shadcn/stories/Input.stories.tsx [hunk 9]
```diff
@@ -1,103 +1,27 @@
...
-import { Input } from '../ui/input'
-import { Label } from '../ui/label'
-import { toast } from '../ui/use-toast'
-
-const FormSchema = z.object({
-  username: z.string().min(2, {
-    message: 'Username must be at least 2 characters.',
-  }),
-})
+import { InputOTP, InputOTPGroup, InputOTPSeparator, InputOTPSlot } from '../ui/input-otp'
```
Comment (MildTomato) [inline packages/ui/src/components/shadcn/stories/Input.stories.tsx:2]: you've overridden this story

File: packages/ui/tailwind.config.js [hunk 10]
```diff
@@ -17,10 +17,15 @@ module.exports = config({
...
         },
+        'caret-blink': {
+          '0%,70%,100%': { opacity: '1' },
+          '20%,50%': { opacity: '0' },
+        },
       },
       animation: {
         'flash-code': 'flash-code 1s forwards',
         'flash-code-slow': 'flash-code 2s forwards',
+        'caret-blink': 'caret-blink 1.25s ease-out infinite',
```
[the remark this answers is not in the document]
Comment (MildTomato) [reply packages/ui/tailwind.config.js:28]: ahh sorry, this is for UI component isn't it

File: packages/ui/tailwind.config.js [hunk 11]
```diff
@@ -17,10 +17,15 @@ module.exports = config({
           '0%': { backgroundColor: 'rgba(63, 207, 142, 0.1)' },
           '100%': { backgroundColor: 'transparent' },
         },
+        'caret-blink': {
```
[the remark this answers is not in the document]
Comment (MildTomato) [reply packages/ui/tailwind.config.js:20]: nvm, my mistake

File: packages/ui/tailwind.config.js (hunk 10 above)
[the remark this answers is not in the document]
Comment (fsansalvadore) [reply packages/ui/tailwind.config.js:28]: same

File: packages/ui/src/layout/banners/LW11CountdownBanner/Countdown.tsx [hunk 12]
```diff
@@ -0,0 +1,34 @@
+import Countdown from 'react-countdown'
```
Comment (MildTomato) [inline packages/ui/src/layout/banners/LW11CountdownBanner/Countdown.tsx:1]: not a blocker but UI patterns should probably be not be so specific "LW11CountdownBanner" doesn't sound very reusable

**Review by MildTomato — CHANGES_REQUESTED** [review]
.

File: apps/www/components/LaunchWeek/11/LW11Meetups.tsx [hunk 13]
```diff
@@ -0,0 +1,138 @@
...
+  }, [])
+
+  function handleSelectMeetup(meetup: Meetup) {
+    setActiveMeetup(meetup)
+  }
+
+  if (!isMounted) return null
+
+  return (
+    <div className="max-w-7xl mx-auto grid grid-cols-1 xl:grid-cols-12 gap-8 text-[#EDEDED]">
```
[the remark this answers is not in the document]
Comment (MildTomato) [reply apps/www/components/LaunchWeek/11/LW11Meetups.tsx:83]: could follow up in another PR

File: packages/ui/src/components/shadcn/stories/Input.stories.tsx (hunk 9 above)
Comment (MildTomato) [reply packages/ui/src/components/shadcn/stories/Input.stories.tsx]: you pushed yet?

File: packages/ui/src/layout/banners/LW11CountdownBanner/Countdown.tsx (hunk 12 above)
Comment (fsansalvadore) [reply packages/ui/src/layout/banners/LW11CountdownBanner/Countdown.tsx:1]: yeah can fix in a follow up pr

**Review by MildTomato — APPROVED** [review]

**Pull request merged**